### ISSUE TITLE
Princeton Only MVWs aren't indexing into DPUL.

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/dpul#1300]
### Summary

As per https://github.com/pulibrary/dpul/blob/main/architecture-decisions/0007-indexing-visibility.md we've decided to index princeton-only material into DPUL. However, multi volume works (such as https://figgy.princeton.edu/catalog/d2d488af-2980-4554-a9ef-aacbaf463ec8) do not index. It may be that child manifests aren't getting the token appended to them.

### Impact 
There are no hard deadlines.

### Priority recommendation

- [ ] asap
- [ ] within the next 3 weeks
- [x] PO will prioritize

### Bug Reproduction

In the console you can run `IIIFResource.where(url: "https://figgy.princeton.edu/concern/scanned_resources/d2d488af-2980-4554-a9ef-aacbaf463ec8/manifest").last.solr_documents` to reproduce. You should get solr documents - instead you get errors.

Example: 

```ruby
irb(main):009:0> IIIFResource.where(url: "https://figgy.princeton.edu/concern/scanned_resources/d2d488af-2980-4554-a9ef-aacbaf463ec8/manifest").last.solr_documents
Called with #<Spotlight::Etl::Context:0x000055f79bbc68b0 @arguments=[#<IIIFResource id: 44296, exhibit_id: 15...
Failed to get https://figgy.princeton.edu/concern/scanned_resources/7d37f047-f46f-459b-bbc4-c8f4588ef95d/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0cbc05af-4336-4438-8cab-4f24ee9ab8b6/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/bb41df1f-049f-43e5-81f7-5bb1643165b2/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/a9bf13b5-10be-42fa-8468-bc9ca5ca0587/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/df00932a-8b79-4476-aba0-748d8b0b1fbd/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0f14809b-d5b8-4bf7-a52b-2cef676705d5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/835a04f5-2df4-4667-9e4b-b24f519d5b69/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/3c525352-5855-4f98-aacf-6bf413c3808e/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/7c910d5e-183a-445c-b686-ec3e7c79eb97/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/f1ed9302-9a0e-4510-a027-d1d205fe7902/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/3e277ec2-3f4f-475e-8f0e-eb04c4829fc5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/69e8aec4-c5bb-4cad-a480-0f0184fb7c96/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/a4611ed3-93fd-450d-97fd-e4671e2b84da/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/56d8a29f-e519-475c-b698-3fe9dd1ca884/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d343c618-2184-430b-941c-11efff129883/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0c405d95-2c4f-4c60-83fb-b4fc820097a4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/b7cf438a-541d-4efc-8010-861f6574414b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/ba7d1564-d522-47fe-8dff-c5ed98174448/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/71fbe22a-f404-4e5b-ae59-576934c5f854/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/713020a7-2a26-48d3-8cd3-d80665b25fff/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/32f26e06-da6d-4c83-a5a6-f21bac1fd503/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/37ac6f28-ac18-4c28-9222-1ac0a4dd4286/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/1504e58c-43da-4f6d-ae65-f7004be6dac7/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/128a33f3-02e2-4751-a430-103f891864d5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/f506cea2-2d59-4eff-8a4f-cdd4d269b01b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/3e0c4b66-35a8-4771-8044-6da0d83f55c5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2b1da44c-fb71-4678-b6d1-0367f31bc479/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/a1164d9b-f7a2-4a46-ae30-6a586d5c34b5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/140e8ac8-5fc1-4a87-b9a2-e9d1411baf7d/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/f97d7911-e82c-42c9-9e57-4194cb8cee7f/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/be6e9069-e363-40f0-bf1a-aca091b95d57/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0019bf17-79fb-4e96-88db-6068f092c816/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/ef4c3f71-59d0-4a41-8e0a-b22a30d6a05b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/9b8d7ad2-2e25-4ee9-be25-a31591643444/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d0951aaf-9ed8-4086-b618-3e2b7bfcdc12/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e6241a83-317b-48df-87f5-74bb80b126ba/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0e7b2572-75e5-46f0-83f7-d210241e4310/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/373ecb94-bf12-4002-bc8e-0f9ab3967aae/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/09bfbe20-4b8e-43d9-9c25-d2113e7b6a39/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/c10c4509-67ff-462e-a0b0-51f3b9e77ca5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/37694785-f662-4980-ac0b-d21a9ccf3d27/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/89d6af8b-38f3-4fb2-964c-de6f6104db1a/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/06bbded3-9ffd-48f1-a9b1-a12edc191519/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/12536333-1c88-428b-91b9-83582bb2ff63/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/4b835e5e-57d8-4901-a398-570b7c1acea6/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/9a4bf986-d911-4dca-8be0-49e9fd144703/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/176b4c0e-d244-4daf-8bce-187e13b320f9/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/eaca0ecf-db34-42ed-87b2-bca43fe8af15/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/ae9f6499-91f6-4112-bae8-4d7533f07cf0/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/13da8f4c-2d86-4619-8949-74e94f76f22d/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/912e53a9-aa2e-4ac6-8492-238489ed62a3/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/26916fee-968a-46d9-987f-52f7558164b8/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/6355a31e-b974-46cf-9c11-5403ef608d08/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0e58c92c-37e1-4399-b64a-9510ac93ffe7/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/64f36d78-179d-47e1-8cbe-cd3a0cdb4eb1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2ca16dd8-4535-4944-80d0-ced8c8076c05/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/fd8613b5-bbd9-4348-9eeb-7877e4f4687b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/8f351987-9656-481a-b33e-3234d92415ab/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e96cbc5e-d7af-413b-bba8-9ee0ee12a45a/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0fc128f4-68a3-4fbe-8680-08b3d36a8ce3/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0d414d12-d81b-49c4-abb6-cc646e5d01af/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/173c33ff-3887-4679-9883-2e0b59d7ec99/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/29262eed-52bf-434b-85fc-7d8de442abd5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/4d60c543-d6f5-4e8c-87fc-72c8bcaecac4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/97e043b4-3910-428e-9c15-bad751beafbb/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/730d1137-f951-4ecd-8533-bb8cb0b2c38c/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/7a51b252-3d0f-4363-9cc2-ab8d2fcf3003/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e67b4fd5-ed19-40f4-af5b-36ba2451be2d/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/38c92a84-0215-438a-9208-0b39260e34f1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2941c012-c121-484e-9819-948f952fc6ed/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/33c4716e-3dd4-4e2e-938e-fc61e19b7513/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e79b1935-e4c4-4ae3-8cd4-85bc2e804edc/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/5e5cac66-eb4a-49b3-a0f1-af541bd9deed/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2278fb7b-fe2a-4f9d-a011-2853863fa415/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/730950de-74af-4bac-98ae-b8bd8bd2058c/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/3b512f73-825e-4589-8b0f-8eff4667dd4c/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d7e67f32-2ab7-47da-b092-d7c9cd803d7a/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d153f8fb-880c-441c-b599-7e8e0303ae0f/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2df7fcd7-cf9a-4c00-b502-d8292a74212c/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/42bee95e-b1b4-4a44-b828-f69ba37320a4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d53e7357-ea25-4f27-a26f-f24a2968477b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/5f460935-6ded-45d2-a43c-82a61abd45c5/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/bf95d706-1718-4198-bebc-d993559d7689/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/f0a3d45b-c2eb-4063-aef8-ff70307a97cd/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/8b5f2b37-f358-4f72-80b1-96edb7ebe061/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/62239301-7d99-45b3-8c5b-d5321e5a4c56/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/8afd87f4-cea2-474d-a61a-a51b085b9fd1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d5176b38-7944-4172-988f-ba347936fe62/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/739645f6-db3f-4fa7-ba34-9fd52c8ae712/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/bd7df7e8-a68a-4d2f-8544-338a11541744/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/bd976524-e045-43c6-a492-464235a36029/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/cd5ca621-f9d4-4a23-a825-474719824ea8/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/ed9e9142-0cd0-4351-b46f-695f527436ff/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/1a513416-6d93-4015-9e27-2a9b9d50edb1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/9a86de67-5bb0-4c9a-89d8-38cf7221cc4e/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/83a0af5e-9497-4409-839f-e3d2c72ec345/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/68266b2b-4401-4e72-a953-156d3fc2395a/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/8ab6aec7-f3e1-43a7-a3d5-d100f659ba54/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/d99c2a09-c2e2-485a-bff3-7d84e615e781/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/b74ff8f8-14ef-40a7-8ddf-f9a21f48fbd2/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/aee84919-3e68-4750-8548-3d30b1b7ed90/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/c5d11a48-9a91-49ab-b390-223aaff8c249/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/a1d67d57-39ae-4ab3-82df-3f17d497cc86/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e9fde3a1-c667-45ae-bb9c-a3d3eda7c2c1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/02de3d00-00db-4ca2-9710-a4e0010a433c/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/85d36261-48b8-4859-a68d-64f7fd00e4f0/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/1367ffb7-381e-443c-b3e9-b6f5718ae742/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/921eb6e4-7ac9-4ba9-8d31-a5156f277da6/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/c83364e3-9701-4908-bb6d-49fb959823f4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/2180f241-9d48-4d75-aec6-6e391d791132/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/54fcf0e3-3960-4fd0-ba43-bdfd2d01abca/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/fc536b83-4597-46db-9418-2b4e3b97d969/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/269d48be-ac41-4201-9fc1-be7cc9ece3a1/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/02b0172c-b9c6-4e49-a338-09dee10d8b22/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/8549e890-fab6-4997-8cea-e2b6243d447b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/c7f447c7-ec89-4f58-9c70-330d74815eb4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/0d0cab0c-0466-4747-b91a-48a4a6b5613b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/3df40be7-930a-4fec-ab3a-292e2172fe37/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/706f15a6-d81e-42b7-b5bf-4a5a1f2bb987/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/ddd45228-da5f-45a9-b11e-c71b7142ee6e/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/4a562f3c-11b4-472f-b84f-0c060a884da4/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/667e7d21-f872-408b-9313-b1bc01c29098/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/50a13fdf-5f58-443a-964b-02af18ce5ae7/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/c1b2bd31-c0d2-4aa1-8502-9759952c5db3/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/859a8be9-a5ff-4d2b-8253-82cb4ed99570/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/71e77d8f-bbbf-4252-b52f-8428bdd2f1ea/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/214b396f-33e1-4136-a4a5-a61094e4a4bf/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/abfca0cd-4d19-4618-b2e4-442f78120df2/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/10afbe1e-2fea-4c81-8208-4ff97f57c736/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/a31fbd88-e7cc-4f6f-8531-9d0194017764/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/b11f3c46-7dae-4ca9-9501-820b53485831/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/b395079e-09aa-4981-96e3-b13328fb2e7b/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/bb9acd02-d8b1-4ceb-8c84-20cdde30e3ad/manifest
Failed to get https://figgy.princeton.edu/concern/scanned_resources/e11a1c22-9cc3-4513-8233-0a021e6cdf5e/manifest
Called with #<Spotlight::Etl::Context:0x000055f79bbc68b0 @arguments=[#<IIIFResource id: 44296, exhibit_id: 15...
Called with id []
```

**Comment by hackartisan** [comment]
Here's the honeybadger error https://app.honeybadger.io/projects/51733/faults/85405934

**Comment by eliotjordan** [comment]
Closed by #1440 

1. `IIIFResource.where(url: "https://figgy.princeton.edu/concern/scanned_resources/d2d488af-2980-4554-a9ef-aacbaf463ec8/manifest").last.solr_documents` does not return an error.
1. Princeton Slavic collection re-indexed successfully. 
![Screenshot 2023-04-07 at 9 43 29 AM](https://user-images.githubusercontent.com/784196/230656258-7db35d5e-baf8-4710-9dc5-6199b2dc35ad.png)
